specification/attestation/resource-manager/readme.python.md

```diff
@@ -18,10 +18,10 @@ python:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/attestation/azure-mgmt-attestation/azure/mgmt/attestation
+  output-folder: $(python-sdks-folder)/azure-mgmt-attestation/azure/mgmt/attestation
```

Comment:
This is not the standard path, why would you change it this way?

Reply:
What is the standard path?